src/editor/EditorStatusBar.js

```diff
@@ -465,6 +465,17 @@ define(function (require, exports, module) {
             var document = EditorManager.getActiveEditor().document,
                 fullPath = document.file.fullPath;
 
+            var fileType = (document.file instanceof InMemoryFile) ? "newFile" : "existingFile",
+                filelanguageName = lang ? lang._name : "";
+
+            HealthLogger.sendAnalyticsData(
+                "usagelanguageChange" + filelanguageName + fileType,
+                "usage",
```

Comment:
@niteskum can all analytics strings be moved to another file as variables and use that variable everywhere needed?

Reply:
moved all common strings constant to HealthLogger.js as earlier all constants were defined in this file